reactfrontend/src/components/addAssetForm/addAssetForm.js

```diff
@@ -9,7 +9,10 @@ class AddAssetForm extends Component {
         assetTag: '',
         assetType: 'Laptop',
         createdBy: '',
+        assetStatus: 'Ready',
+        serialNumber: '',
       },
+      message: '',
       company: "",
       showMessage: false,
       companyusers: "",
```

Comment:
Do certain fields require a default value while others not? If this is the case, is an approach other than hardcoding feasible?

Reply:
How a question that is not answered marked as resolved?

Reply:
Only the ones that have a dropdown, it is so because the dropdown always shows a value even when the user did not select anything